Sources/StreamChat/ChatClient.swift

```diff
@@ -363,6 +358,11 @@ public class _ChatClient<ExtraData: ExtraDataTypes> {
     /// are received.
     public func disconnect() {
         clientUpdater.disconnect()
+        
+        backgroundTaskScheduler?.startListeningForAppStateUpdates(
```

Comment:
Not sure if it's ok or we better add a `BackgroundTaskScheduler.stopListeningForAppStateUpdates` and incapsulate it in there

Reply:
`stopListeningForAppStateUpdates` would be a good addition